content/docs/03-ai-sdk-core/55-testing.mdx

```diff
@@ -0,0 +1,125 @@
+---
+title: Testing
+description: Learn how to use AI SDK Core mock providers for testing.
+---
+
+# Testing
+
+Testing language models can be challenging, because they are non-deterministic
+and calling them is slow and expensive.
+
+To enable you to unit test your code that uses the AI SDK, the AI SDK Core
+includes a mock providers and test helpers that you can use to test your code.
```

Comment:
consider rewriting
"AI SDK Core includes a mock provider and test helpers to allow you to unit test your application."